src/amo/css/AddonReview.scss

```diff
@@ -0,0 +1,52 @@
+@import "~core/css/inc/vars";
+
+.AddonReview {
+  text-align: center;
+
+  h2 {
```

Comment:
It's better for performance and if you nest another component under this that has a `p`, `h2` or `textarea` then you won't get the wrong styles applied. I find the sass nesting to be hard to read and I think the added verbosity (which seems minimal to me) is worth reducing the nesting.

It also means we have one way to do things. Wrapping raw tags in a parent class only works on leaf components but applying a single class to the thing you want to style will work anywhere.